pkg/tsdb/cloudwatch/metrics.go

```diff
@@ -362,6 +362,20 @@ var metricsMap = map[string][]string{
 	"AWS/NetworkFirewall":   {"DroppedPackets", "Packets", "PassedPackets", "ReceivedPacketCount"},
 	"AWS/OpsWorks":          {"cpu_idle", "cpu_nice", "cpu_steal", "cpu_system", "cpu_user", "cpu_waitio", "load_1", "load_15", "load_5", "memory_buffers", "memory_cached", "memory_free", "memory_swap", "memory_total", "memory_used", "procs"},
 	"AWS/Polly":             {"2XXCount", "4XXCount", "5XXCount", "RequestCharacters", "ResponseLatency"},
+	"AWS/PrivateLinkEndpoints": {
+		"ActiveConnections",
+		"BytesProcessed",
+		"NewConnections",
+		"PacketsDropped",
+		"RstPacketsReceived",
+	},
```

Comment:
I didn't want to do this because it is the longest string in the map. But sure ;-) I have applied both your sugestions and executed gofmt on the file.

Reply:
Oh yes, I see! 
I'm glad you did not the first time 😄  Made reviewing much easier!